_posts/2020-05-26-how-to-pair-sucessfully.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "How to Pair Successfully"
+date: 2020-05-19 12:30:00
+categories: ["learning", "pairing"]
+author: fiondl
+---
+
+Since starting at OmbuLabs as the Junior Developer I have had the incredible opportunity to pair with my teammates. A lot. Over the months we have begun to develop better practices and routines to make the most of our pairing sessions, and now I would like to share some of our practices and tips for making a pairing session successful. At OmbuLabs we are a fully remote company so all of our pairing sessions are done through Tuple or Zoom, but this advice could translate easily to in person sessions as well.
+
+<!--more-->
+
+##When and How Long to Pair For
```

Comment:
@FionaDL Title won't work without the space (https://github.com/ombulabs/blog/blob/e509174365d7ffc4d84b69dacef9e5716940b135/_posts/2020-05-26-how-to-pair-sucessfully.markdown)

```suggestion
## When and How Long to Pair For
```